<?xml version="1.0" ?>
<launch>
  <!-- params -->
  <param name="forest_file" value="$(find leg_tracker)/config/trained_leg_detector_res=0.33.yaml" />
  <param name="scan_topic" value="/scan" />
  <param name="fixed_frame" value="map"/>
  <!--<param name="publish_people_frame" value="map"/> -->  
  <param name="scan_frequency" value="15"/>  
  <param name="dist_travelled_together_to_initiate_leg_pair" value="0.3"/>  
  <param name="cluster_dist_euclid" value="0.05"/>  

  <!-- run detect_leg_clusters -->  
  <node pkg="leg_tracker" type="detect_leg_clusters" name="detect_leg_clusters" output="screen"/>

  <!-- run joint_leg_tracker -->
  <node pkg="leg_tracker" type="joint_leg_tracker.py" name="joint_leg_tracker" output="screen"/>
 
   <!-- run local_occupancy_grid_mapping -->
  <node pkg="leg_tracker" type="local_occupancy_grid_mapping" name="local_occupancy_grid_mapping" output="screen"/> 
   
</launch>

